src/Components/test/E2ETest/Infrastructure/SeleniumStandaloneServer.cs

```diff
@@ -123,7 +123,7 @@ void LogOutput(object sender, DataReceivedEventArgs e)
                     output = builder.ToString();
                 }
 
-                throw new InvalidOperationException($"Failed to start selenium sever. {Environment.NewLine}{output}", ex.GetBaseException());
+                throw new InvalidOperationException($"Failed to start selenium sever. {System.Environment.NewLine}{output}", ex.GetBaseException());
```

Comment:
The fact that it's necessary to qualify all uses of `Environment` makes me wonder if this is going to bother customers, too. Should we consider renaming the `Microsoft.AspNetCore.Components.Environment` namespace?